atom/browser/api/atom_api_tray.cc

```diff
@@ -60,6 +60,14 @@ void Tray::OnBalloonClosed() {
   Emit("balloon-closed");
 }
 
+void Tray::OnRightClicked(const gfx::Rect& bounds) {
+  Emit("right-clicked", bounds);
+}
+
+void Tray::OnDragFiles(const std::vector<std::string>& files) {
+  Emit("drag-files", files);
```

Comment:
As stated by DOM's docs:

> The drag event is fired when an element or text selection is being dragged (every few hundred milliseconds)
> The drop event is fired when an element or text selection is dropped on a valid drop target.

`drop` should be better than `drag` in this case?